src/Battery.vala

```diff
@@ -65,5 +212,45 @@ namespace Power {
 
             return path;
         }
+
+        public string get_info () {
+            var percent = (int)Math.round (percentage);
+            if (percent <= 0) {
+              return _("Calculating…");
+            }
+
+            if (percent == 100) {
+              return _("Fully charged");
+            }
+
+            var info = _("%i%% charged").printf (percent);
+            return info;
+        }
+
+        public string get_health () {
+            var capacity = (int)Math.round (capacity);
+            if (capacity <= 60) {
+              return _("Poor");
+            }
+
+            if (capacity > 60) {
+              return _("Normal");
+            }
+
+            if (capacity > 90) {
+              return _("Excelent");
+            }
+
+            return _("Unknown");
+        }
```

Comment:
I get that it might make sense to abstract this here, but in this case I wonder if more precise data is more useful? Perhaps we could display both the string and percent, e.g. `Poor (59%)`?

Also open to thoughts from @elementary/ux

Reply:
D'oh! It would help if I read the rest of the code and tested before commenting. I see you have the "maximum capacity" below this which makes sense. It can be separate if we want, I'm indifferent. I do think it's kind of nice to give a string like you have, too, but maybe we do some research on the values and provide a couple more levels, e.g. excellent, good, fair, poor.